docs/guides/authentication.mdx

```diff
@@ -0,0 +1,11 @@
+---
+id: authentication
+title: Authentication & Login
+---
+
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
+
+<DocCardList items={useCurrentSidebarCategory().items}/>
```

Comment:
@vinckr this is an easy way to create overview pages. It's obviously better to not auto-generate them, but it helps with one issue:

If the `sidebar_label` is set in a document, it overrides the sidebar label defined in the config. So while the sidebar config says the label is `foo`, if the label in the doc is `bar`, then the label is `bar`. This is problematic for categories with a link:

```
    {
      type: "category",
      label: "Identity management",
      collapsed: false,
      collapsible: false,
      link: {
        type: "doc",
        id: "kratos/manage-identities/overview", // if this has `sidebar_label: foo`, then the category will be called "foo"
      },
```

For this reason, these auto-generated categories pages work as fillers.

Reply:
lgtm